recipes/glacier_lengths/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = 'glacier_lengths' %}
+{% set version = '0.0.4' %}
+
+package:
+  name: '{{ name | lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/erikmannerfelt/glacier_lengths/archive/refs/tags/v0.0.4.tar.gz
```

Comment:
can you use `https://github.com/erikmannerfelt/glacier_lengths/archive/refs/tags/v{{ version }}.tar.gz`? That way automatic updates from the bots will work fine.

Reply:
Changed in 2469ca2!